test/fuzz/headers.dict

```diff
@@ -0,0 +1,4 @@
+# Dictionary to populate HeaderValueOptions
+"path"
```

Comment:
This should be `:path` and `:method` below.

Reply:
thanks! I added a few more. While I think these special header values will eventually be found via coverage guidance because they're defined in header files, I think it's useful to speed up.